compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs

```diff
@@ -1138,7 +1162,7 @@ internalTypes = mkUniqSet
     ]
 
 desugarTypes :: UniqSet FastString
-desugarTypes = mkUniqSet ["Consuming", "PreConsuming", "PostConsuming", "NonConsuming", "Implements"]
+desugarTypes = mkUniqSet ["Consuming", "PreConsuming", "PostConsuming", "NonConsuming"]
```

Comment:
I think you can add `"Method"` in here. We don't really want `DA.Internal.Desugar.Method` ending up in LF.

Reply:
Oh, and I think `"HasMethod"` too.

Reply:
ah well spotted, thanks!